### PR TITLE
Speech v1 GAPIC.

### PR DESCRIPTION
**Pull request by lukesneeringer**

### CODE REVIEW

**Comment by shinfan** [comment]
LGTM, let @geigerj approve as well.

**Comment by lukesneeringer** [comment]
Just merged googleapis/toolkit#1152 so we could get tests also. :-)

File: generated/python/gapic-google-cloud-speech-v1/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+googleapis-common-protos>=1.5.2, <2.0dev
+google-gax>=0.15.7, <0.16dev
```
Comment (geigerj) [inline generated/python/gapic-google-cloud-speech-v1/requirements.txt]: 0.15.8?

Comment (lukesneeringer) [inline generated/python/gapic-google-cloud-speech-v1/requirements.txt]: The changes in 0.15.8 did not require upgrades from downstream clients.

**Pull request merged**